### ISSUE TITLE
Compatibility with prettier 3.0.0

### ISSUE THREAD

**Issue by mpflanzer** [markw65/prettier-plugin-monkeyc#3]
Hi,
 Thanks for the plugin! While I was searching for something to format Monkey C code I didn't expect to find anything but even got a plugin for prettier.

I noticed that the plugin currently doesn't seem to work with prettier version 3. Do you have plans to upgrade it?

For instance I get this error:

```
[error] source/OTView.mc: TypeError: estree_print is not a function
[error]     at Object.printAst [as print] (/home/moritz/repos/tools/prettier-plugin-monkeyc/build/prettier-plugin-monkeyc.cjs:12089:20)
[error]     at callPluginPrintFunction (file:///home/moritz/repos/chickadee-engineering/o-technics/node_modules/prettier/index.mjs:18140:20)
[error]     at printAstToDoc (file:///home/moritz/repos/chickadee-engineering/o-technics/node_modules/prettier/index.mjs:18095:22)
[error]     at async coreFormat (file:///home/moritz/repos/chickadee-engineering/o-technics/node_modules/prettier/index.mjs:18405:14)
[error]     at async formatWithCursor (file:///home/moritz/repos/chickadee-engineering/o-technics/node_modules/prettier/index.mjs:18603:14)
[error]     at async formatFiles (file:///home/moritz/repos/chickadee-engineering/o-technics/node_modules/prettier/internal/cli.mjs:6778:18)
[error]     at async main (file:///home/moritz/repos/chickadee-engineering/o-technics/node_modules/prettier/internal/cli.mjs:6970:5)
[error]     at async Module.run (file:///home/moritz/repos/chickadee-engineering/o-technics/node_modules/prettier/internal/cli.mjs:6924:5)
```

**Comment by markw65** [comment]
I hadn't tried it with 3.0. I'll look into it!

**Comment by markw65** [comment]
Should be fixed by d0929bfc1dae5444c1276ca7ffe58d087ff9220b, and in `@markw65/prettier-plugin-monkeyc@1.0.50`.

I've not updated [the vscode extension](https://marketplace.visualstudio.com/items?itemName=markw65.prettier-extension-monkeyc&ssr=false#overview) yet, because I don't think it should be affected (the vscode prettier extension still ships with prettier@2.x).

**Comment by mpflanzer** [comment]
Super cool. Thanks a lot for the quick fix!
